source/cloud/cloud-administration/sso-office.md

```diff
@@ -3,36 +3,79 @@ ___
 
    ![](../../../source/images/AzureApp_SetupMenuv2.png)
 
-Now the App Registration has been created and you can configure it further. The standard Azure AD documentation is [here](https://docs.microsoft.com/en-gb/azure/active-directory/develop/quickstart-register-app) for reference.
+Once the App Registration has been created, you can configure it further. The standard Azure AD documentation is [here](https://docs.microsoft.com/en-gb/azure/active-directory/develop/quickstart-register-app) for reference.
 
-4. Select **Certificates and Secrets** from the menu, and click the button to generate a **New Client secret**. Provide a description and define the expiry for the token. 
+### Step 2: Generate a new client secret in Azure Portal
 
-   ![](../../../source/images/AzureApp_Client_Secret_Expiry.png)
+1. In the Azure Portal, select **Certificates and Secrets** from the menu, then select the button to generate a **New Client secret**. 
+
+2. Provide a description, define the expiry for the token, then select **Add**.
 
-Click _Add_ and you will be provided with the _client secret value_, copy this and save it for use in the Mattermost configuration as the **Application Secret Password**.
+   ![](../../../source/images/AzureApp_Client_Secret_Expiry.png)
 
-5. Select **Overview** from the menu and copy the _Application (client) ID_ and the _Directory (tenant) ID_, for use in the Mattermost configuration as the **Application ID** and as part of the **Auth Endpoint** and **Token Endpoint** URL.
+In Azure Portal, select **Overview** from the menu, then copy and paste both the _Application (client) ID_ and the _Directory (tenant) ID_ to a temporary location. You will enter these values as an **Application ID** and as part of an **Auth Endpoint** and **Token Endpoint** URL in the Mattermost System Console.
 
    ![](../../../source/images/AzureApp_App_Directory_IDsv2.png)
 
-6. Log in to Mattermost and then go to the **System Console > OAuth 2.0 > Select OAuth 2.0 service provider**, choose **Office 365** as the service provider. Enter the _client secret value_ you copied in Step 4 as the **Application Secret Password**. Paste the _Application (client) ID_ you saved in Step 5 into the **Application ID** field.
+### Step 3: Configure Mattermost for Office 365 SSO
+
+1. Log in to Mattermost, then go to the **System Console > Authentication > OpenID Connect**.
+
+2. Select **Office 365** as the service provider.
+
+3. Paste in the _Directory (tenant) ID_ from Azure Portal as the **Directory (tenant) ID** in Mattermost.
+
+4. The **Discovery Endpoint** for OpenID Connect with Office 365 is prepopulated with ``https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration``.
+
+5. Paste in the _Application (client) ID_ from Azure Portal as the **Client ID** in Mattermost.
+
+6. Paste in the _client secret value_ from Azure Portal as the **Client Secret** in Mattermost.
+
+6. Open the Mattermost `config.json` file.
```

Comment:
This would be for manual update. I don't think steps 6 and 7 belong here.

Reply:
@sbishel - I was confused by this too. This is pre-existing content in the docs, and while it feels like the manual way to edit directly via the config.json file, it also read like a couple of additional manual steps that are required specifically for Office 365. Can you confirm either way, please? If everything beyond the "open the MM config.json file" step is no longer needed for Office 365, I'll happily take it out!

Reply:
Yep, re-reviewed and confirmed the two steps should be removed. This is no longer necessary. I looked at the current documentation and it appears out of date.